package.nix

```diff
@@ -0,0 +1,131 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  makeWrapper,
+  e2fsprogs-nofortify,
+  jefferson,
+  lz4,
+  lziprecover,
+  lzop,
+  p7zip,
+  nix-filter,
+  sasquatch,
+  sasquatch-v4be,
+  simg2img,
+  ubi_reader,
+  unar,
+  partclone,
+  zstd,
+  versionCheckHook,
+  rustPlatform,
+}:
+
+let
+  # These dependencies are only added to PATH
+  runtimeDeps = [
+    e2fsprogs-nofortify
+    jefferson
+    lziprecover
+    lzop
+    p7zip
+    sasquatch
+    sasquatch-v4be
+    ubi_reader
+    partclone
```

Comment:
Ideally the whole file should be committed as part of the nix commit, but then the partclone addition at this line should be part of commit b21b6dc291583af6b7ec9b7c3d63ee8302328841. This way we separate the nix rework from the addition of partclone to it.